app/templates/components/large-search.hbs

```diff
@@ -68,5 +68,7 @@
     {{/search-result-set}}
   {{else if search.isRunning}}
     {{dot-spinner}}
+  {{else if (and search.performCount query.length)}}
```

Comment:
I think this will erroneously show when a search has happened and a query has been typed, but it is not a valid query that triggers a search. Take a look at `queryIsValid` CP in `app/components/large-search.js`